Source/WPEFramework/Controller.h

```diff
@@ -202,7 +219,9 @@ namespace Plugin {
         };
 
     private:
+        Controller(Controller&&);
```

Comment:
This is, I guess, from the old days, when we did not allow std++11 since the delete keyword did not exist before std++11 we could not use it there. Since we now say that the minimum requirement for Thunder is std++11, we changed the rules and now copy/move constructurs/oassignment operators we do no want, we make them public with the = delete statement. We do not actively update all code to apply these rules, but if we touch the code, we do update it. So would expect these constructors to be moved to public with the = delete; behind it..

Reply:
done